### PR TITLE
Feature/persist presenter info

### PR DESCRIPTION
**Pull request by chenry**
This will now retrieve the persenters from the database.

### CODE REVIEW

File: scripts/dummy_data.sql [hunk 1]
```diff
@@ -14,9 +14,12 @@ INSERT INTO conf_admin.room(
     select 'Short Description', 'Full Description', conference.id
                from conf_admin.conference where conference.name = 'Sample Conference';
 
+insert into conf_admin.company
+(name, short_desc, full_desc, logo_path) values ('GFS', 'Gordon Food Service', 'Food distribution', 'http://logo_path/gfs.png');
+
 INSERT INTO conf_admin.presenter(
             user_id, short_bio, job_title, company_affiliation_id)
-    VALUES (null, 'short bio', 'sample title', null);
+    VALUES (null, 'short bio', 'sample title', 1);
```
Comment (will1771) [inline scripts/dummy_data.sql:22]: I think the company_affiliation_id should be set to something like "where name = 'GFS'" so you can blow away the tables and run the script again without resetting sequences.

File: src/main/java/com/ioextendedgr/web/repository/PresenterRepository.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.ioextendedgr.web.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.ioextendedgr.web.data.Presenter;
+
+/**
+ * Created by Scott Williams on 3/4/2015.
```
Comment (will1771) [inline src/main/java/com/ioextendedgr/web/repository/PresenterRepository.java:8]: LIES

File: scripts/dummy_data.sql (hunk 1 above)
Comment (chenry) [inline scripts/dummy_data.sql:22]: Great point. Will try to look at this again tonight

File: src/main/java/com/ioextendedgr/web/repository/PresenterRepository.java (hunk 2 above)
Comment (chenry) [inline src/main/java/com/ioextendedgr/web/repository/PresenterRepository.java:8]: Ha. That is what I get from the copy paste.

File: scripts/dummy_data.sql (hunk 1 above)
Comment (chenry) [inline scripts/dummy_data.sql:22]: I am not going to implement this change.  please understand that this dummy_data.sql is only used for testing and nothing more.

**Pull request merged**